docs/core/extensions/snippets/configuration/console-json/TransientFaultHandlingOptions.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace ConsoleJson.Example
 {
-    public class TransientFaultHandlingOptions
+    public record TransientFaultHandlingOptions
```

Comment:
Looks like the code is correct but the docs wording should be changed to "class".

@IEvangelist Can you please take a look since this is related to https://github.com/dotnet/docs/pull/22210?

Reply:
Actually there are two different articles using the same code snippet. I'm not sure if both should be classes or not.

Reply:
All options pattern should be shown using a `class`, the `record` will only work with rare-undesired workarounds and partially. It was a mistake to use `record` with this.

Reply:
Article 1:

https://github.com/dotnet/docs/blob/master/docs/core/extensions/options.md
https://docs.microsoft.com/en-us/dotnet/core/extensions/options

Article 2:

https://github.com/dotnet/docs/blob/master/docs/core/extensions/configuration-providers.md
https://docs.microsoft.com/en-us/dotnet/core/extensions/configuration-providers